app/signin/page.tsx

```diff
@@ -0,0 +1,134 @@
+"use client";
+import React, { useState } from "react";
+import styles from "../../styles/SigninPage.module.css";
+
+import { loginSuccess } from "../../utils/commonUtil";
+import { postReqeustApi } from "../../utils/requestApi";
+import SignInfoInputComponent from "../../components/SignInfoInputComponent";
+
+// email 정규식
+const regex = new RegExp("[a-z0-9]+@[a-z]+.[a-z]{2,3}");
+
+const SigninPage = () => {
+  const [signinInfo, setSigninInfo] = useState({
+    email: false,
+    password: false,
+  });
+
+  const [emailErrorText, setEmailErrorText] = useState("");
+  const [pwErrorText, setPwErrorText] = useState("");
+
+  const validCheck = (name: string, value: string) => {
+    let isValid = false;
+    if (name === "email") {
+      if (value === "") {
+        // 오류
+        setEmailErrorText("이메일을 입력해 주세요.");
+      } else if (!regex.test(value)) {
+        // 오류
+        setEmailErrorText("올바른 이메일 주소가 아닙니다.");
+      } else {
+        // 정상
+        setEmailErrorText("");
+        isValid = true;
+      }
+    } else if (name === "password") {
+      if (value === "") {
+        setPwErrorText("비밀번호를 입력해 주세요.");
+      } else {
+        setPwErrorText("");
+        isValid = true;
+      }
+    }
+    return isValid;
+  };
+
+  const changeSigninInfo = (e: any) => {
+    const isValid = validCheck(e.target.name, e.target.value);
+
+    setSigninInfo({
+      ...signinInfo,
+      [e.target.name]: isValid,
+    });
```

Comment:
상태값을 업데이트할 때, 이전 상태값을 이용하는 경우에는 콜백 함수(리액트에서는 updater function이라고 표현)를 이용해서 업데이트를 해주는게 좋습니다!
```suggestion
    setSigninInfo(prev => ({
      ...prev,
      [e.target.name]: isValid,
    }));
```